src/main/java/com/google/sps/data/friend_map/UserFriendsMap.java

```diff
@@ -20,49 +20,41 @@
 import java.util.Map;
 import java.util.HashMap;
 import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableMap;
 
-/*
-* A map of all the direct friendships between users
+/**
+* <p>A map of all the direct friendships between users which will be used
```

Comment:
This is only for subsequent paragraphs. The first paragraph doesn't need the `<p>`

Reply:
Got it!